pkg/operator/apiserver/controller/apiservice/apiservice_controller_test.go

```diff
@@ -412,6 +437,89 @@ func TestDisabledAPIService(t *testing.T) {
 
 }
 
+func TestPreconditionsForEnabledAPIServices(t *testing.T) {
+	var (
+		bootstrapComplete = &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{Name: "bootstrap", Namespace: "kube-system"},
+			Data:       map[string]string{"status": "complete"},
+		}
+
+		bootstrapNoStatus = &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{Name: "bootstrap", Namespace: "kube-system"},
+			Data:       map[string]string{"noStatus": "complete"},
```

Comment:
Is "noStatus" an arbitrary non-"status" key?

Reply:
yes.